modules/__tests__/matchPattern-test.js

```diff
@@ -12,7 +12,7 @@ describe('matchPattern', function () {
   }
 
   it('works without params', function () {
-    assertMatch('/', '/path', 'path', [], [])
+    assertMatch('/', '/path', '/path', [], [])
```

Comment:
This test case covers a purely internal function – `matchRoutes` is not part of our public API.